go/store/prolly/map.go

```diff
@@ -233,6 +233,44 @@ func (m Map) IterRange(ctx context.Context, rng Range) (MapIter, error) {
 	return filteredIter{iter: iter, rng: rng}, nil
 }
 
+// GetOrdinal returns the smallest ordinal position at which the key >= |query|.
+func (m Map) GetOrdinal(ctx context.Context, query val.Tuple) (uint64, error) {
```

Comment:
name this something more descriptive like `GetOrdinalForKey()`. I would expect `GetOrdinal()` to take an integer `i` and returns the i'th tuple in the Map.